models/repo/release.go

```diff
@@ -334,6 +335,20 @@ func (s releaseMetaSearch) Less(i, j int) bool {
 	return s.ID[i] < s.ID[j]
 }
 
+// InitReleaseRepo makes sure the Repo field of releases is not nil
+func InitReleaseRepo(ctx context.Context, rels ...*Release) error {
+	var err error
+	for _, release := range rels {
```

Comment:
In most situations, all releases have the same repository. So we should cache repository here.